src/Features/Core/Portable/QuickInfo/CommonSemanticQuickInfoProvider.cs

```diff
@@ -209,6 +209,18 @@ void AddSection(string kind, ImmutableArray<TaggedText> taggedParts)
                 AddSection(QuickInfoSectionKinds.DocumentationComments, documentationContent);
             }
 
+            var returnsDocumentationContent = GetReturnsDocumentationContent(symbols, groups, semanticModel, token, formatter, syntaxFactsService, cancellationToken);
+            if (!returnsDocumentationContent.IsDefaultOrEmpty)
+            {
+                var builder = ImmutableArray.CreateBuilder<TaggedText>();
+                builder.AddLineBreak();
+                builder.AddText(FeaturesResources.Returns_colon);
+                builder.AddLineBreak();
+                builder.Add(new TaggedText(TextTags.Space, "  "));
```

Comment:
If Returns is multiple lines, is this going to only mean the first line is indented versus the left margin of the entire section being indented?

Reply:
I'll fix this for the next submission and include a screenshot.